simtools/tests/test_visualize.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+
+import logging
+import os
+import sys
+import numpy as np
+from astropy import units as u
+from astropy.io import ascii
+from .. import visualize
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+
+def test_plot_1D():
+
+    logger.debug('Testing plot1D')
+
+    xTitle = 'Wavelength [nm]'
+    yTitle = 'Mirror reflectivity [%]'
+    headersType = {'names': (xTitle, yTitle),
+                   'formats': ('f8', 'f8')}
+    title = 'Test 1D plot'
+
+    dataIn = np.loadtxt('tests/testData/ref_200_1100_190211a.dat',
+                        usecols=(0, 1), dtype=headersType)
+
+    # Change y-axis to percent
+    if '%' in yTitle:
+        if np.max(dataIn[yTitle]) <= 1:
+            dataIn[yTitle] = 100*dataIn[yTitle]
```

Comment:
Pep8: space in between operators

Reply:
Do you mean the *? 
PEP8 recommends adding space around operators with lowest priorities, but there's no such case here. I prefer to have multiplications without spaces around them.

Reply:
Yes, the *
Ok, I though we were always suppose to add spaces. It's fine then.